nucypher/cli/painting/staking.py

```diff
@@ -103,7 +107,6 @@ def prettify_stake(stake, index: int = None) -> str:
 
     pretty = f'| {index if index is not None else "-"} ' \
              f'| {stake.staker_address[:6]} ' \
-             f'| {stake.worker_address[:6]} ' \
```

Comment:
Totally agree, it doesn't have sense to show the worker here, since all stakes have the same.